Source/WebKit/UIProcess/API/gtk/WebKitWebViewBase.cpp

```diff
@@ -419,6 +420,14 @@ webkitWebViewBaseAccessibleInterfaceInit(GtkAccessibleInterface* iface)
 {
     iface->get_first_accessible_child = webkitWebViewBaseAccessibleGetFirstAccessibleChild;
 }
+#else
+static void
+webkitWebViewBaseAccessibleInterfaceInit(GtkAccessibleInterface* iface)
+{
+    UNUSED_PARAM(iface);
```

Comment:
You can just omit the parameter name:

```suggestion
webkitWebViewBaseAccessibleInterfaceInit(GtkAccessibleInterface*)
{
```

(Source/WebKit is built with -Wno-unused-parameter, so even that is not strictly necessary, but best to write the code as if it is used.)